docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -31,15 +31,86 @@ NOTE: The data store that you choose must be a centralized system that is access
+    ...
+    <external-data-store name="my-mysql-database">
+        <class-name>com.hazelcast.datastore.JdbcDataStoreFactory</class-name>
+        <properties>
+            <property name="jdbcUrl">jdbc:mysql://mysql.example.org:3306</property>
+            <property name="username">my_user</property>
+            <property name="password">my_password</property>
+        </properties>
+        <shared>true</shared>
+    </external-data-store>
+    <external-data-store name="my-other-database">
+        <class-name>com.hazelcast.datastore.JdbcDataStoreFactory</class-name>
+        <properties>
+            <property name="jdbcUrl">jdbc:h2:mem:my-other-database</property>
+        </properties>
+        <shared>true</shared>
+    </external-data-store>
+    ...
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  external-data-store:
+    my-mysql-database:
+      class-name: com.hazelcast.datastore.JdbcDataStoreFactory
+      properties:
+        jdbcUrl: jdbc:mysql://mysql.example.org:3306
+        username: my_user
+        password: my_password
+      shared: true
+    my-other-database:
+      class-name: com.hazelcast.datastore.JdbcDataStoreFactory
+      properties:
+        jdbcUrl: jdbc:h2:mem:my-other-database
+      shared: true
+----
+====
 
+The external data store has the following configuration elements:
+
+* `name` (required): The unique identifier for the external data store.
+* `class-name` (required): The name of the `com.hazelcast.datastore.ExternalDataStoreFactory` implementation that will be used to create instances of the data store.
+* `properties`: The configuration properties for an implementation of the external data store factory. See xref:#built-in-data-store-factories[Built-in Data Store Factories] for properties supported by specific implementations.
+* `shared`: Specifies whether the external data store factory returns reusable data stores for different Jet jobs and `MapStores` and `MapLoaders`. Specific behaviour depends on the implementation of the factory. The default value is `true`.
+
+You can also add new external data stores dynamically at runtime, see xref:configuration:dynamic-config.adoc[dynamic configuration].
+
+==== Built-in Data Store Factories
+
+* `com.hazelcast.datastore.JdbcDataStoreFactory` creates JDBC-based external data stores. This implementation is based on link:https://github.com/brettwooldridge/HikariCP[HikariDataSource]. All properties are passed directly to `HikariConfig`. For available options see link:https://github.com/brettwooldridge/HikariCP#gear-configuration-knobs-baby[HikariCP configuration].
+If there are more JDBC connections used on a single member from a single job, they will share the same data store and connection pool.
+
+* For other types of data store, you need to implement `com.hazelcast.datastore.ExternalDataStoreFactory`.
+
+== Loading Data From Your External Store
+The MapLoader interface includes the following features for loading data.
```

Comment:
Just need to add a carriage return, (extra line) between the heading and the first line of text.

Reply:
Fixed: https://github.com/hazelcast/hz-docs/pull/514/commits/b42ceaf46f9243a3d62a34aee5f55cc0a66c0f4a